File: 240402/Date to Date/date-to-date.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int month1 = sc.nextInt();
        int day1 = sc.nextInt();
        int month2 = sc.nextInt();
        int day2 = sc.nextInt();

        int elapsedDate = 1;
        
        int dateArr[] = new int[]{0,31,28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int date = dateArr[month1];
        while(true) {
            if(day1 == day2 && month1 == month2) 
                break;
            
            elapsedDate++;
            day1++;
            if(day1 > dateArr[month1]) {
                month1++;
                day1 = 1;
            }
            
        }
        System.out.println(elapsedDate);
    }
}

File: 240418/Date to Date/date-to-date.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)

    val m1 = sc.nextInt()
    val d1 = sc.nextInt()
    val m2 = sc.nextInt()
    val d2 = sc.nextInt()

    val result1 = (1..m1).map { calculateMonthOfDay(it) }.sum() + d1
    val result2 = (1..m2).map { calculateMonthOfDay(it) }.sum() + d2
    
    print(result2 - result1)
}

fun calculateMonthOfDay(n: Int): Int {
    return when(n) {
        1 -> 31
        2 -> 28
        3 -> 31
        4 -> 30
        5 -> 31
        6 -> 30
        7 -> 31
        8 -> 31
        9 -> 30
        10 -> 31
        11 -> 30
        else -> 31
    }
}